Help command broken on servers

Need to investigate

/help
[20:25:42] [Server thread/INFO]: An unknown error occurred while attempting to perform this command
[20:25:42] [Server thread/ERROR]: Couldn't process command: 'help'
java.lang.NullPointerException
        at shukaro.warptheory.handlers.WarpCommand.func_71519_b(WarpCommand.java:74) ~[WarpCommand.class:?]
        at net.minecraft.command.CommandHandler.func_71557_a(CommandHandler.java:219) ~[z.class:?]
        at net.minecraft.command.CommandHelp.func_71534_d(SourceFile:83) ~[au.class:?]
        at net.minecraft.command.CommandHelp.func_71515_b(SourceFile:38) ~[au.class:?]
        at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:94) [z.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71333_ah(DedicatedServer.java:370) [lt.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:335) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]

@draknyte1 I am trying to build WarpTheory but getting an error with Gradle.

C:\Users\hendric\Documents\repos\WarpTheory>gradle setupDecompWorkspace
```

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'WarpTheory'.
> ForgeGradle 1.2 does not support forge 1.7.10

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.011 secs

C:\Users\hendric\Documents\repos\WarpTheory>
```

--- a/src/main/java/com/matthewprenger/helpfixer/HelpFixer.java
+++ b/src/main/java/com/matthewprenger/helpfixer/HelpFixer.java
@@ -9,8 +9,8 @@ import java.util.List;
 import javax.annotation.Nonnull;
 
 import net.minecraft.command.CommandBase;
-import net.minecraft.command.CommandHelp;
 import net.minecraft.command.CommandException;
+import net.minecraft.command.CommandHelp;
 import net.minecraft.command.ICommand;
 import net.minecraft.command.ICommandSender;
 import net.minecraft.server.MinecraftServer;
@@ -45,23 +45,23 @@ public class HelpFixer {
                     try {
                         if (command.getCommandName() == null) {
                             FMLLog.warning(
-                                String.format(
-                                    "[HelpFixer] Identified command with null name, Ignoring: %s",
-                                    command.getClass().getName()));
+                                    String.format(
+                                            "[HelpFixer] Identified command with null name, Ignoring: %s",
+                                            command.getClass().getName()));
                             iterator.remove();
                         } else if (command.getCommandUsage(sender) == null) {
                             FMLLog.warning(
-                                String.format(
-                                    "[HelpFixer] Identified command with null usage, Ignoring: %s",
-                                    command.getClass().getName()));
+                                    String.format(
+                                            "[HelpFixer] Identified command with null usage, Ignoring: %s",
+                                            command.getClass().getName()));
                             iterator.remove();
                         }
                     } catch (CommandException commandException) {
                         FMLLog.warning(
-                            String.format(
-                                "[HelpFixer] Command to throw exception %s, Ignoring: %s",
-                                commandException.getClass().getName(),
-                                command.getClass().getName()));
+                                String.format(
+                                        "[HelpFixer] Command to throw exception %s, Ignoring: %s",
+                                        commandException.getClass().getName(),
+                                        command.getClass().getName()));
                         iterator.remove();
                     }
                 }

--- a/src/main/java/com/matthewprenger/helpfixer/HelpFixer.java
+++ b/src/main/java/com/matthewprenger/helpfixer/HelpFixer.java
@@ -10,6 +10,7 @@ import javax.annotation.Nonnull;
 
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.CommandHelp;
+import net.minecraft.command.CommandException;
 import net.minecraft.command.ICommand;
 import net.minecraft.command.ICommandSender;
 import net.minecraft.server.MinecraftServer;
@@ -41,17 +42,26 @@ public class HelpFixer {
                 Iterator<ICommand> iterator = list.iterator();
                 while (iterator.hasNext()) {
                     ICommand command = iterator.next();
-                    if (command.getCommandName() == null) {
-                        FMLLog.warning(
+                    try {
+                        if (command.getCommandName() == null) {
+                            FMLLog.warning(
                                 String.format(
-                                        "[HelpFixer] Identified command with null name, Ignoring: %s",
-                                        command.getClass().getName()));
-                        iterator.remove();
-                    } else if (command.getCommandUsage(sender) == null) {
-                        FMLLog.warning(
+                                    "[HelpFixer] Identified command with null name, Ignoring: %s",
+                                    command.getClass().getName()));
+                            iterator.remove();
+                        } else if (command.getCommandUsage(sender) == null) {
+                            FMLLog.warning(
                                 String.format(
-                                        "[HelpFixer] Identified command with null usage, Ignoring: %s",
-                                        command.getClass().getName()));
+                                    "[HelpFixer] Identified command with null usage, Ignoring: %s",
+                                    command.getClass().getName()));
+                            iterator.remove();
+                        }
+                    } catch (CommandException commandException) {
+                        FMLLog.warning(
+                            String.format(
+                                "[HelpFixer] Command to throw exception %s, Ignoring: %s",
+                                commandException.getClass().getName(),
+                                command.getClass().getName()));
                         iterator.remove();
                     }
                 }

